src/main/kotlin/com/team4099/robot2023/subsystems/intake/IntakeIO.kt

```diff
@@ -0,0 +1,111 @@
+package com.team4099.robot2023.subsystems.intake
+
+import org.littletonrobotics.junction.LogTable
+import org.littletonrobotics.junction.inputs.LoggableInputs
+import org.team4099.lib.units.base.amps
+import org.team4099.lib.units.base.celsius
+import org.team4099.lib.units.base.inAmperes
+import org.team4099.lib.units.base.inCelsius
+import org.team4099.lib.units.derived.Angle
+import org.team4099.lib.units.derived.degrees
+import org.team4099.lib.units.derived.inDegrees
+import org.team4099.lib.units.inDegreesPerSecond
+import org.team4099.lib.units.perSecond
+
+interface IntakeIO {
+  class IntakeIOInputs : LoggableInputs {
+    var leaderArmPosition: Angle = 0.0.degrees
+    var leaderArmVelocity = 0.0.degrees.perSecond
+
+    var leaderArmAppliedOutput = 0.0
+    var followerArmAppliedOutput = 0.0
```

Comment:
There is no more follower on the intake (it is one motor for pivot, one motor for rollers)